client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,153 @@
+
+function getAmountAsFormattedString( amount ) {
+	// Takes a Number, formats it to 2 decimal places, and prepends a "$".
+	// This mimics the existing behaviour. I'm assuming we only view/report
+	// on earnings in USD.
+	const formattedAmount = '$' + numberFormat( amount, 2 );
+	// Differs from previous behaviour in that we don't want "$0.00" as a result.
+	// Per design spec we'll return "$0" in this scenario.
+	// https://github.com/Automattic/wp-calypso/issues/72045
+	return formattedAmount === '$0.00' ? '$0' : formattedAmount;
+}
+
+function getHighlights( earnings ) {
+	const total = earnings && earnings.total_earnings ? Number( earnings.total_earnings ) : 0;
+	const owed = earnings && earnings.total_amount_owed ? Number( earnings.total_amount_owed ) : 0;
+	const paid = total - owed;
+
+	const highlights = [
+		{
+			heading: translate( 'Earnings', { comment: 'Total WordAds earnings to date' } ),
+			amount: total,
+		},
+		{
+			heading: translate( 'Paid', {
+				comment: 'Total WordAds earnings that have been paid out',
+			} ),
+			amount: paid,
+		},
+		{
+			heading: translate( 'Outstanding amount', {
+				comment: 'Total WordAds earnings currently unpaid',
+			} ),
+			amount: owed,
+		},
+	];
+	// Transform/index the data for use with React.
+	return highlights.map( ( highlight, i ) => ( {
+		id: i,
+		formattedAmount: getAmountAsFormattedString( highlight.amount ),
+		icon: <Icon icon={ starEmpty } />,
+		...highlight,
+	} ) );
+}
+
+function payoutNotices( earnings ) {
+	const owed =
+		earnings && earnings.total_amount_owed ? numberFormat( earnings.total_amount_owed, 2 ) : '0.00';
+	const notice = {
+		id: 'notice',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s does not exceed the minimum $100 needed to make the payment.',
+			{
+				comment: 'WordAds: Insufficient balance for payout.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	const limit = {
+		id: 'limit',
+		value: translate(
+			'Payment will be made as soon as the total outstanding amount has reached $100.',
+			{
+				comment: 'WordAds: Payout limit description.',
+			}
+		),
+	};
+	const payout = {
+		id: 'payout',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s will be paid approximately 45 days following the end of the month in which it was earned.',
+			{
+				comment: 'WordAds: Payout will proceed.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	return owed < 100 ? [ notice, limit ] : [ payout ];
+}
+
+function HighlightsSectionHeader( props ) {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const iconReference = useRef( null );
+	const localizedTitle = translate( 'Totals', {
+		comment: 'Heading for WordAds earnings highlights section',
+	} );
+	if ( ! props.notices || props.notices.length === 0 ) {
+		return <h1 className="highlight-cards-heading">{ localizedTitle }</h1>;
+	}
+	return (
+		<h1 className="highlight-cards-heading">
+			{ localizedTitle }{ ' ' }
+			<Gridicon
```

Comment:
Let's try to use `<Icon />` instead of a `<Gridicon />`. We are migrating as many icons as possible to help with loading them for Odyssey Stats.

Reply:
Thanks for the heads up. I wasn't aware of this. I'll get this fixed up!